flutter/lib/src/integrations/app_start/app_start_tracker.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+
+import '../../../sentry_flutter.dart';
+
+@internal
+class AppStartInfo {
+  final DateTime start;
+  final DateTime end;
+  final SentryMeasurement measurement;
+
+  AppStartInfo(this.start, this.end, this.measurement);
+}
+
+@internal
+class AppStartTracker {
+  static final AppStartTracker _instance = AppStartTracker._internal();
+  factory AppStartTracker() => _instance;
+  AppStartTracker._internal();
```

Comment:
```suggestion
  AppStartTracker._();
```
Let's be consistent with other classes we already have, like [HubAdapter](https://github.com/getsentry/sentry-dart/blob/b0811ccc76dccb91004b36b5f710163daf733d92/dart/lib/src/hub_adapter.dart#L18) and [NoOp classes](https://github.com/getsentry/sentry-dart/blob/b0811ccc76dccb91004b36b5f710163daf733d92/dart/lib/src/noop_client.dart#L7)
This applies for other places in this PR as well